codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -348,25 +350,6 @@ public void close(ChannelHandlerContext ctx, ChannelPromise promise) throws Exce
         }
     }
 
-    public void deregister(ChannelHandlerContext ctx, ChannelPromise promise) throws Exception {
```

Comment:
Well bcz I suck at merging/cherry-picking of course :) 4408180d.  I also removed them when cherry-picking this back to 4.1 (double oops) :) https://github.com/netty/netty/pull/3703. You guys are running 4.1 right? Submit PRs against 4.1 or risk me mucking up the merge :)